circle.yml

```diff
@@ -11,7 +9,18 @@ test: &TEST
     - checkout
     - run:
         name: Install Julia
-        command: brew cask install julia
+        command: |
+          curl -s -L --retry 7 -o julia.dmg \
+            "https://julialang-s3.julialang.org/bin/mac/x64/$JULIA_VER/julia-$JULIA_VER-latest-mac64.dmg"
+          mkdir jlmnt
+          hdiutil mount -readonly -mountpoint jlmnt julia.dmg
+          cp -a jlmnt/*.app/Contents/Resources/julia ~/
+          echo 'PATH=$HOME/bin:$PATH' >> $BASH_ENV
```

Comment:
I got this from https://discuss.circleci.com/t/how-to-add-a-path-to-path-in-circle-2-0/11554/10, but it doesn't appear to be working properly, hence the failures on Circle.

Reply:
You need to `source` bashrc, like in that link

Reply:
But also I think you could either just not use $PATH, or do an install to whatever directory brew is using

Reply:
Actually you need that to be `echo 'export PATH=$HOME/bin:$PATH' >> $BASH_ENV

The export is important

Reply:
Whoooooops. Added `export` and `source`, but either I'm still derping or modifying the path isn't the right solution.

Previously I tried symlinking the executable to `/usr/local/bin` but it couldn't find it there, and I tried symlinking to `/usr/bin` but got a permissions error, even with `sudo`.

Reply:
Might as well just reference Julia by path then?